### ISSUE TITLE
Cloning Window element is broken

### ISSUE THREAD

**Issue by AlaaSarhan** [jdittrich/quickMockup#67]
**Is**
- Cloning a window element just breaks everything .. like in this screenshot
![image](https://user-images.githubusercontent.com/1851753/71118085-aea5ad00-21d7-11ea-85a8-ac1ed419609e.png)
- Assigned children inside the container are not cloned

**Should**
- Cloning a window element should just create another proper window element as expected.. looking like the following
![image](https://user-images.githubusercontent.com/1851753/71118182-d8f76a80-21d7-11ea-935b-52214521f23f.png)
- Assigned children inside the container should be cloned too

**Notes**
* `selectedFigure.clone()` does not seem to work just out-of-the-box when cloning a figure that have "real" children (that is Figure children in draw2d definition) such as the "Window" label that is inside a window figure

**Comment by jdittrich** [comment]
> another related question I had when testing this: should cloning a Container element also clone its "children"

Yes – this is what I would expect there and what typical graphics applications do, too.

**Comment by AlaaSarhan** [comment]
> > another related question I had when testing this: should cloning a Container element also clone its "children"
> 
> Yes – this is what I would expect there and what typical graphics applications do, too.

Updated description with that
